code/controllers/subsystem/radiation.dm

```diff
@@ -57,6 +59,18 @@ SUBSYSTEM_DEF(radiation)
 		if (current_insulation <= pulse_information.threshold)
 			continue
 
+		if(isnull(pulse_information.max_range) && pulse_information.intensity) // If max range is undefined, then set it to the distance where the intensity drops to the constant.
```

Comment:
I'm not a fan of this. Intensity is a metric with no obvious meaning, I intentionally removed it with the new system.

It's not clear to feature coders or maintainers what an "intensity" number actually means, at least not without looking everywhere else for context. It is obvious what max range is.

Reply:
Could remove that if you don't want intensity to be used like that.